content/millennium/r4/scheduling/slot.md

```diff
@@ -193,6 +193,64 @@ List an individual Slot by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Patch
+
+<%= beta_tag(known_issues: ["A 500 Internal Server Error is thrown when an invalid version id is used in the If-Match Header"]) %>
```

Comment:
this is just the action right? action: true

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/c64ba7144dedbc115bfc60b9a5a0b7044c08a9ec)

Reply:
nit: can we get a space after action:

Reply:
[added](https://github.com/cerner/fhir.cerner.com/pull/608/commits/8cdb6007af44fcd1b69e95c59520aa58d66d8b09)